data/_nav.yml

```diff
@@ -215,23 +221,30 @@ reference:
         - url: /docs/reference/experimental.html
           title: Experimental API Markers
 
-    - title: Core Libraries
+    - title: Core Libraries Reference
       content:
         - url: /api/latest/jvm/stdlib/index.html
           title: Standard Library
         - url: /api/latest/kotlin.test/index.html
           title: kotlin.test
 
-    - title: Reference
+    - title: Language Reference
       description: Grammar and other information
       content:
         - url: /docs/reference/keyword-reference.html
           title: Keywords and Operators
         - url: /docs/reference/grammar.html
           title: Grammar
 
+    - title: Multiplatform Programming
+      description: Reusing code across JVM, JS, and Native
+      content:
+        - url: /docs/reference/platform-specific-declarations.html
+          title: Platform-Specific Declarations
+        - url: /docs/reference/building-mpp-with-gradle.html
+          title: Building with Gradle
 
-    - title: Java Interop
+    - title: Java
```

Comment:
I'm afraid that a user can't guess what this title means. Java is a programming language (so is JavaScript), and this is a Kotlin website...